bing_bert/deepspeed_train.py

```diff
@@ -230,7 +238,8 @@ def train(args,
 def update_learning_rate(args, config, current_global_step, optimizer):
     global last_global_step_from_restore
 
-    global_step_for_lr = current_global_step - last_global_step_from_restore
+    #global_step_for_lr = current_global_step - last_global_step_from_restore

```

Comment:
This will break backwards-compatibility since it previously worked for both bing and nvidia datasets. Can you explain why this change is needed?